whois-commons/src/test/java/net/ripe/db/whois/common/support/WhoisClientHandler.java

```diff
@@ -85,8 +82,8 @@ public ChannelFuture disconnect() {
 
     public ChannelFuture sendLine(String query) throws InterruptedException {
         Assert.assertTrue(success);
-        ChannelFuture future = channel.write(query + "\n");
-        success = future.await(3, TimeUnit.SECONDS);
+        ChannelFuture future = channel.writeAndFlush(query + "\n");
```

Comment:
Changed behaviour from write() to writeAndFlush() - is this necessary?

Reply:
Yes, it is. As noted in my other response here https://github.com/RIPE-NCC/whois/pull/754#discussion_r573160493